detect_secrets/core/potential_secret.py

```diff
@@ -113,6 +115,12 @@ def json(self) -> Dict[str, Union[str, int, bool]]:
         if hasattr(self, 'is_secret') and self.is_secret is not None:
             attributes['is_secret'] = self.is_secret
 
+        if hasattr(self, 'is_added') and self.is_added is not None:
+            attributes['is_added'] = self.is_added
+
+        if hasattr(self, 'is_removed') and self.is_removed is not None:
+            attributes['is_removed'] = self.is_removed
+
         return attributes
 
     def __eq__(self, other: Any) -> bool:
```

Comment:
might worth fixing here also the line 112:
` if hasattr(self, 'line_number') and self.line_number:`
if line is 0 it will not be updated.

Reply:
I think that if 0 = false